ui/src/router/components/configuration/components/pyfunc_config_section/PyFuncConfigViewGroup.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { EuiFlexGroup, EuiFlexItem } from "@elastic/eui";
+import { ConfigSectionPanel } from "../../../../../components/config_section";
+import { ContainerConfigTable } from "../docker_config_section/ContainerConfigTable";
+import { ResourcesConfigTable } from "../ResourcesConfigTable";
+import { ConfigMultiSectionPanel } from "../../../../../components/config_multi_section_panel/ConfigMultiSectionPanel";
+import { PyFuncRefConfigTable } from "./PyFuncRefConfigTable";
+
+export const PyFuncConfigViewGroup = ({
+  componentName,
+  pyfuncConfig,
+  dockerConfig,
+}) => {
+  const items = [
+    {
+      title: "Pyfunc Ensembler Details",
+      children: <PyFuncRefConfigTable config={pyfuncConfig} />,
+    },
+  ];
+
+  if (!!dockerConfig) {
+    items.push({
```

Comment:
This ensures that the router details screen is minimally able to display a Pyfunc ensembler's name and ID (but not the `dockerConfig` related stuff) should the image building job fail during router deployment.